source/type/pb_converter.cpp

```diff
@@ -639,6 +639,15 @@ void fill_pb_object(const nt::ValidityPattern* vp, const nt::Data&,
     validity_pattern->set_days(vp->days.to_string());
 }
 
+void fill_pb_object(const nt::MetaVehicleJourney* nav_mvj,
```

Comment:
Usually yes, but we don't see the need for it in a close future so I vote to let it that way.